src/CodeGenerator/InjectorGenerator.php

```diff
@@ -16,13 +19,13 @@
 use Zend\Di\Resolver\DependencyResolverInterface;
 
 /**
- * Generator for the depenendency injector
+ * Generator for the dependency injector
  *
  * Generates a Injector class that will use a generated factory for a requested
  * type, if available. This factory will contained pre-resolved dependencies
  * from the provided configuration, definition and resolver instances.
  */
-class InjectorGenerator
+class InjectorGenerator implements LoggerAwareInterface
```

Comment:
Same as above: implementing this interface is a mistake. Add an optional ctor argument instead